release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -404,6 +404,50 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
 [id="ocp-4-16-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-16-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.12.0
+* Metrics Server to 0.7.1
+* node-exporter to 1.8.0
+* Prometheus to 2.52.0
+* Prometheus Operator to 0.73.2
+* Thanos to 0.35.0
+
+[id="ocp-4-16-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
```

Comment:
Remove *New* as updates to existing rules is unavailable.

Reply:
Could the note come after the bullet points? The note did not prepare me for the added rules. Maybe afterwards would give customers a moment to orientate themselves with the updates.

Reply:
This, and the "The in-cluster monitoring stack for this release includes the following new and modified features." semtence is a template we have been using in Monitoring RNs for all the existing doc versions. I will consult with @bburt-rh if we really want to deviate from that, thank you!

Reply:
@eromanova97 @dfitzmau - My take is that it is fine as-is to stay consistent with the way the monitoring RNs have been structured since at least the 4.9 release. The same applies to the previous comment.